build.gradle

```diff
@@ -45,4 +45,9 @@ ext {
     assertjVersion = '2.8.0'
     supportTestVersion = '1.0.1'
     espressoVersion = '3.0.1'
+
+    // According to https://github.com/googlesamples/android-custom-lint-rules/tree/master/android-studio-3
+    // the lint version should match to the used Android Gradle Plugin by the formula "AGP Version X.Y.Z + 23.0.0"
+    // E.g. "AGP Version 3.1.3 + 23.0.0 = Lint Version 26.1.3"
+    lintVersion = '26.1.3'
```

Comment:
We could also calculate this by ourselfs, right? 🤔 
Just extract the AGP version and "add" the values to 23.0.0 ...

Reply:
Yes, we can calculate this ourself. But I would prefer to change that still manually:
- It is stated "should" and so the actual version might differ from the formula
- Google recently just decoupled the Support Lib Versions and so it might be that this version gets uncoupled from the build tools version aswell